contracts/strategies/mstable/IdleMStableStrategy.sol

```diff
@@ -163,8 +183,8 @@ contract IdleMStableStrategy is Initializable, OwnableUpgradeable, ERC20Upgradea
 
     /// @notice Internal function to deposit the underlying tokens to the vault
     /// @param _amount amount of tokens to deposit
-    /// @return number of reward tokens minted
-    function _depositToVault(uint256 _amount) internal returns (uint256) {
+    /// @return _minted number of reward tokens minted
+    function _depositToVault(uint256 _amount) internal returns (uint256 _minted) {
         underlyingToken.approve(address(imUSD), _amount);
```

Comment:
we can store `imUSD` in a local variable (there are 3 reads to it in this method)